### PR TITLE
Withdrawal changes

### PR DESCRIPTION
**Pull request by 4855062503**
Will continue to update today

### CODE REVIEW

File: lib/withdrawal.dart [hunk 1]
```diff
@@ -363,6 +367,8 @@ class _WithdrawalFormScreenState extends State<WithdrawalFormScreen> {
                               validator: (value) {
                                 if (value == null || value.isEmpty)
                                   return 'Please enter a value';
+                                value = value.trim();
+                                Clipboard.setData(ClipboardData(text: value));
```
Comment (djpnewton) [inline lib/withdrawal.dart:371]: why are you copying to the clipboard here?

**Review by djpnewton — CHANGES_REQUESTED** [review]
deposit screens have cancel buttons, I guess we should do the same here

File: lib/withdrawal.dart (hunk 1 above)
Comment (4855062503) [reply lib/withdrawal.dart:371]: Just realized I forgot to remove it. Was doing it for debugging to see if it was really an empty string or not.

**Comment by djpnewton** [comment]
need to merge from master and fix conflicts

**Comment by djpnewton** [comment]
probably want cancel buttons like the deposit screens

**Pull request merged**